### PR TITLE
Added first two donations by WISH Pool

### PR DESCRIPTION
**Pull request by bclens**
Added new WISH Pool folder, donations.yml and related screenshots for donations made

### CODE REVIEW

File: WISH/donations.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+- ticker: WISH
+  recipient: 
+    name: Akaraka Limited
+    address: 77 Robinson Road, #13-00, Robinson 77, Singapore 068896
+    contact: info@akaraka.org.sg
+    ada: 32.26
```
Comment (kiriakos) [inline WISH/donations.yml:6]: The `ada` and `usd` props are indented too much by one level making the CI throw up, can you fix that?

**Comment by bclens** [comment]
Hi,

I have already removed the spaces in front of the Ada and Usd entries but I got another mail saying it failed again.  Anything else I need to fix?

Pls kindly advise

**Comment by kiriakos** [comment]
Ah! The recipient objects have no `tel` field in them... all fields are mandatory. Can you dig up phone numbers on those charities? 
image

**Comment by bclens** [comment]
Added a telephone number for Akaraka I found from an old web link.  Hope this is ok now

**Review by kiriakos — APPROVED** [review]
LGTM

**Pull request merged**